docs/API.md

```diff
@@ -173,7 +173,7 @@ __Parameters__
 | Param  | Type  | Description  |
 |---|---|---|
 | `bucketName`  |  _string_ | Name of the bucket.  |
-| `callback(err, exists)`  | _function_  | `exists` is a boolean which indicates whether `bucketName` exists or not. `err` is set when an error occurs during the operation. |
+| `callback(err)`  | _function_  | `err` is `null` if the bucket exists. If no callback is passed, a `Promise` is returned. |
```

Comment:
Thanks for the conribution @barinbritva! I think the previous version of this part of the documentation is fine due to the changes that were made in #666. However, in `src/main/minio.js`, we need to update the comments that describe the function.

Reply:
Ok, I see that changes will come in version 5.

Reply:
@barinbritva Are you planning to revert this change to bucketExists()?